.github/workflows/terragrunt-apply-core.yml

```diff
@@ -0,0 +1,83 @@
+name: "Terragrunt apply core"
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+    paths:
+      - ".github/workflows/terragrunt-apply-core.yml"
+      - "terragrunt/aws/core_account/**"
+      - "terragrunt/env/core/**"
+      - "terragrunt/env/common/**"
+      - "terragrunt/env/terragrunt.hcl"
+
+env:
+  AWS_REGION: ca-central-1
+  CONFTEST_VERSION: 0.27.0
+  TERRAFORM_VERSION: 1.1.4
+  TERRAGRUNT_VERSION: 0.36.0
+  TF_VAR_cbs_principal_arn: ${{ secrets.CBS_PRINCIPAL_ARN }}
```

Comment:
You can get rid of this secret - we only need this secret for the Central account.